ramalama/model.py

```diff
@@ -128,14 +128,10 @@ def quadlet(self, args, exec_args):
 [Container]
 Device=+/dev/dri
```

Comment:
We should only include these if they exist on the system I think:

/dev/dri
/dev/kfd

not sure if the container will start if they are specified and they are not present.